### DISCUSSION TITLE
Associate product to a specific stock location

### DISCUSSION THREAD

**Discussion by offlink-io** [medusajs/medusa#4450] [Q&A]
Hi,

I had created a sales channel and two stocks locations are associated on.

![image](https://github.com/medusajs/medusa/assets/128164601/9c89ff96-a2a7-415d-810d-c2edbe65b58e)

Now, I want to create a product and associated it to a specific stock location.

But, when i check the api doc of `Create a product`, nothing about that, only how to set sales channels:

![image](https://github.com/medusajs/medusa/assets/128164601/b77aca15-1773-459a-833f-b8c699270256)

So, how i can set the stock location ?

Thx for your help :)

**Comment by offlink-io** [comment]
Hi,

Any idea ?

thx :)

**Comment by offlink-io** [comment]
Hi,

Any idea ?

thx :)

**Comment by srindom** [comment]
To add a product to a given warehouse you want to go to

- Products > [click a product] > Variants > Click "Manage inventory"

<img width="252" alt="CleanShot 2023-07-11 at 10 51 06@2x" src="https://github.com/medusajs/medusa/assets/7554214/df7a671e-f035-482a-8090-0b95a8ccd917">

- Click "Manage locations"
<img width="733" alt="CleanShot 2023-07-11 at 10 52 10@2x" src="https://github.com/medusajs/medusa/assets/7554214/98bef92a-c53f-4514-a073-19d004d77860">

And then you can configure the locations you want the product to be stocked at.

<img width="721" alt="CleanShot 2023-07-11 at 10 52 21@2x" src="https://github.com/medusajs/medusa/assets/7554214/74a0de3d-7ece-4268-b6ac-e62e364c86c8">
